test/types/objectSearch.test.tsx

```diff
@@ -2,16 +2,52 @@ import faker from 'faker';
 
 import { Tester } from '@mighty-justice/tester';
 
-import { fillInFieldConfig } from '../../src';
+import { fillInFieldConfig, FormCard } from '../../src';
 import ObjectSearch from '../../src/inputs/ObjectSearch';
 
-function getDefaults (overrides?: any) {
-  const fakeLawFirm = () => ({
-      id: faker.random.uuid(),
+function fakeLawFirm () {
+  return {
+    id: faker.random.uuid(),
       name: faker.company.companyName(),
+  };
+}
+
+function getFormDefaults (overrides?: any) {
+  const field = overrides.field || 'law_firm'
+    , endpoint = 'legal-organizations'
+    , type = overrides.type || 'objectSearch'
+    , fieldConfig = fillInFieldConfig({
+      editProps: { debounceWait: 0 },
+      endpoint,
+      field,
+      type,
+      ...overrides.fieldConfig,
     })
-    , field = overrides.field || 'law_firm'
-    , endpoint = overrides.endpoint || 'legal-organizations'
+    , fieldSets = overrides.fieldSets || [[fieldConfig]]
+    , onSave = jest.fn()
+    , model = overrides.model || { law_firm: null }
+    , props = { fieldSets, model, onSave }
+    ;
+
+  return {
+    result: fakeLawFirm(),
+    searchTerm: faker.lorem.sentence(),
+
+    endpoint,
+    field,
+    model,
+    props,
+    type,
+
+    ...overrides,
```

Comment:
shouldn't this go at the end after `fieldConfig`? Also, not sure about the whitespaces here, but don't look good.

Reply:
`overrides.fieldConfig` is already included in the fieldConfig, which is then filled in. So this removes all the defaults from fieldConfig and makes it not filled in... if that makes any sense.